git-branchless-lib/src/core/rewrite/plan.rs

```diff
@@ -196,8 +195,14 @@ impl ToString for RebaseCommand {
                             // Finally, register the new final commit as the
                             // rewritten version of original_commit_oid
                             format!("exec git branchless hook-skip-upstream-applied-commit {original_commit_oid} $(git rev-parse HEAD)")
-                        ];
-                    }
+                        ]
+                    } else {
+                        vec![
+                            // HACK force move branches that used to point at original_commit_oid to new HEAD
+                            // FIXME Yuck! The for loop works by word, not by line; will not work for branches w/ spaces ... is that a thing?
```

Comment:
> will not work for branches w/ spaces ... is that a thing?

No, it's not. At least not w/ `git 2.41`:
```
❯ git br 'foo bar'
fatal: 'foo bar' is not a valid branch name

❯ git br foo\tbar
fatal: 'foo	bar' is not a valid branch name

❯ git br foo\nbar
fatal: 'foo
bar' is not a valid branch name
```

Reply:
TIL